components/molecules/WorkspaceMembersConfig/workspace-members-config.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react";
+import Button from "components/atoms/Button/button";
+import Search from "components/atoms/Search/search";
+import { useToast } from "lib/hooks/useToast";
+import { getAvatarByUsername } from "lib/utils/github";
+import WorkspaceMemberRow from "../WorkspaceMemberRow/workspace-member-row";
+
+interface WorkspaceMembersConfigProps {
+  className?: string;
+  members: WorkspaceMember[];
+  onAddMember: (username: string) => Promise<WorkspaceMember> | undefined;
+  onDeleteMember: (memberId: string) => void;
+  onUpdateMember: (memberId: string, role: WorkspaceMemberRole) => Promise<any>;
+}
+
+const WorkspaceMembersConfig = ({
+  className,
+  members,
+  onAddMember,
+  onDeleteMember,
+  onUpdateMember,
+}: WorkspaceMembersConfigProps) => {
+  const [validInput, setValidInput] = useState(false);
+  const [username, setUsername] = useState("");
+  const [loading, setLoading] = useState(false);
+  const { toast } = useToast();
+
+  const handleChange = async (value: string) => {
+    setUsername(value);
+
+    setValidInput(!!value && value.length > 3);
+  };
+
+  const handleAddMember = async () => {
+    const memberExists = members.find((member) => member.member.login.toLowerCase() === username.toLowerCase());
+
+    if (memberExists) {
+      toast({ description: "Member already exists", variant: "danger" });
+      return;
+    }
+    setLoading(true);
+    const res = await onAddMember(username);
+    setLoading(false);
+    setUsername("");
+    if (res) {
+      toast({ description: "Member added successfully", variant: "success" });
+    } else {
+      toast({ description: "Error: The user entered was not found on OpenSauced!", variant: "danger" });
+    }
+  };
+
+  return (
+    <div className={` ${className && className}`}>
+      <div>
+        <h2 className="flex gap-1 font-medium mb-2 text-md">Workspace Members</h2>
+      </div>
+      <div className="flex items-center gap-5 mt-3">
+        <Search
```

Comment:
The input for adding a new member takes the whole width of the page atm. Can we shrink the size?

Reply:
Yep

Reply:
Update width and button width, height to match others.

<img width="811" alt="image" src="https://github.com/open-sauced/app/assets/42211/e824d509-c94f-4702-8f00-af1feaa76f06">